app/analytics.js

```diff
@@ -14,11 +14,22 @@
  limitations under the License.
  */
 
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+(function() {
+  initGa();
 
-ga('create', 'UA-97952592-1', 'auto');
-// Removes failing protocol check. @see: http://stackoverflow.com/a/22152353/1958200
-ga('set', 'checkProtocolTask', null);
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-97952592-1', 'auto');
+  // Removes failing protocol check. @see: http://stackoverflow.com/a/22152353/1958200
+  ga('set', 'checkProtocolTask', null);
+
+  function initGa() {
```

Comment:
@tabanliviu CONSISTENCY - Function call adds more barriers than it assists with.